### DISCUSSION TITLE
Temperature Range

### DISCUSSION THREAD

**Discussion by PitToad** [rsjaffe/MIDI2LR#721] [Problems]
Is there any way to make the temperature range on a dial match available range in Lightroom?

Lightroom temp range is 2000 - 50,000, Midi2LR allows adustment from 3000-9000

Using 5.4.0.0, with either MidiFighter Twister or Arturia Beatstep, temp range is same on both

**Comment by antzpantz** [comment]
Oddly enough, my Temperature limits, etc are missing from my Options.

Anyway, there's a section on Limits in the wiki that may answer why it's
not possible to adjust those limits:
https://github.com/rsjaffe/MIDI2LR/wiki/Plugin-Options-Dialog#other-settings

**Comment by PitToad** [comment]
Many thanks, literally just stumbled across a Youtube video which I'd paused some time ago, just before the 'other' options were discussed - Mine were set 3k-9k.
Adjusted 2k-50k and it works fine now.
 Not many times I need to fine-tune, it's either loads or none,  so a big range works better for me

16:53 here 
https://www.youtube.com/watch?v=NSQl1txZOmc

**Reply by Gadgetgeek2000** [reply]
Great! I have seen that video before but just recently started looking again at some settings. I was wondering why my temp wouldn't go a high or low as the actual slider in Lightroom allows. I adjusted to encompass the entire range allowed, but then the "click" jumps on the MIDI controller were very imprecise. I prefer the range setup as the default so that I have better control with the knob. If I need more I know it's there in Lightroom, but the usual settings allow a more precise control. 

Thanks! That explained exactly what I experienced.
